content/running-amazon-ec2-workloads-at-scale/deploy_app_codedeploy.md

```diff
@@ -26,93 +27,109 @@ You will now deploy your application to the EC2 instances launched by the Auto S
 3. Take a look at the file on the `Cloud9` editor to confirm the script code has been updated with your Database endpoint. 
 
 4. Then clone the Koel GitHub repo:
+
     ```bash
     cd ~/environment/ec2-spot-workshops/workshops/running-amazon-ec2-workloads-at-scale/
     
     git clone https://github.com/phanan/koel.git
     
     cd koel && git checkout v3.7.2
     ```
-    
+
 {{% notice note %}}
 You'll get an update about being in 'detached HEAD' state. This is normal.
 {{% /notice %}}
 
-5. Next, copy the CodeDeploy configs into the root level of the koel application directory:
+5. Apply fix to **composer.json**:
```

Comment:
Please change this in composer.json file directly. instead of using sed to vfsStream with vfsstream.

Reply:
`composer.json` is stored in different repo and tagged with version.
Details:
In step above we are cloning `https://github.com/phanan/koel.git` repo and switching to `v3.7.2`.